rolling/distribution.yaml

```diff
@@ -7129,7 +7069,6 @@ repositories:
     release:
       packages:
       - turtlebot3_fake_node
-      - turtlebot3_gazebo
```

Comment:
I think this isn't going to work how we want, though maybe @nuclearsandwich can comment.  In particular, the next time someone does a `bloom-release`, I believe that this will "come back".  Maybe the right thing to do here is to blacklist the package in https://github.com/ros2/ros_buildfarm_config/blob/ros2/rolling/release-build.yaml ?  Not sure.